wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -335,6 +342,14 @@ public void checkout(CheckoutRequest request) throws GitException {
         }
         try {
             checkoutCommand.call();
+        } catch(CheckoutConflictException exception){
+            throw new GitConflictException(exception.getMessage(),exception.getConflictingPaths());
```

Comment:
add space between `exception.getMessage(),` and `exception.getConflictingPaths()`